railties/lib/rails/test_help.rb

```diff
@@ -3,13 +3,14 @@
 abort("Abort testing: Your Rails environment is running in production mode!") if Rails.env.production?
 
 require "rails/test_unit/minitest_plugin"
-require 'active_support/testing/autorun'
 require 'active_support/test_case'
 require 'action_controller'
 require 'action_controller/test_case'
 require 'action_dispatch/testing/integration'
 require 'rails/generators/test_case'
 
+require 'active_support/testing/autorun' unless $rails_test_runner
```

Comment:
Yeah, that's better, :+1:. I settled on `run_with_rails_extension`. Thanks, @arthurnn.